kube-client/src/config/mod.rs

```diff
@@ -299,6 +295,8 @@ impl Config {
         let cluster_url = loader
             .cluster
             .server
+            .clone()
+            .unwrap_or_default()
```

Comment:
This should return an error for missing cluster url.

Reply:
Done